discord/cog.py

```diff
@@ -584,9 +584,6 @@ def remove_cog(self, name: str) -> Optional[Cog]:
         if cog is None:
             return
 
-        help_command = self._help_command
-        if help_command and help_command.cog is cog:
-            help_command.cog = None
```

Comment:
Instead of removing this, shouldn't it be placed in an `if hasattr(self, "_help_command")`?

Reply:
It could, although it doesn't exist unless you plan on adding one.